packages/nitro-gui/readme.md

```diff
@@ -10,13 +10,13 @@ Runs the storybook instance for the project, for\
 previewing components in a variety of configured states.
 Open [http://localhost:6006](http://localhost:6006) to view it in the browser.
 
-### `yarn dev`
+### `VITE_RPC_HOST=localhost:4005 yarn dev`
```

Comment:
Do we expect to pass this port with every usage of `yarn dev`? Is it better in something like `.env.development`?

Reply:
Good call [5a3062f](https://github.com/statechannels/go-nitro/pull/1459/commits/5a3062f9dbab3cdd2b490c69f432500f8ac46528)